libnmstate/ifaces/ovs.py

```diff
@@ -163,6 +165,7 @@ class OvsInternalIface(BaseIface):
     def __init__(self, info):
         super().__init__(info)
         self._parent = None
+        self._patch_config = info.get(OVSInterface.PATCH_CONFIG_SUBTREE)
```

Comment:
Please don't store it as it might changed later.

Use a `@property` function.

Reply:
Done, thanks!